test/services/CollectionServiceTest.scala

```diff
@@ -0,0 +1,74 @@
+package services
+
+import com.gu.facia.client.models.{CollectionConfigJson, CollectionJson, ConfigJson, FrontJson, Trail}
+import conf.ApplicationConfiguration
+import org.joda.time.DateTime
+import org.scalatest.{DoNotDiscover, FreeSpec, Matchers}
+import play.api.{Configuration}
+import slices.{Containers, FixedContainers, Story}
+
+@DoNotDiscover class CollectionServiceTest extends FreeSpec with Matchers {
```

Comment:
why the `DoNotDiscover` annotation?

Reply:
Because it was `ctrl-c`'ed from another test class! Whoops! We can remove it.